susemanager-utils/susemanager-sls/src/modules/mgractionchains.py

```diff
@@ -57,22 +58,39 @@ def _get_ac_storage_filenamepath():
 
     return os.path.join(minion_d_dir, '_mgractionchains.conf')
 
+def check_reboot_required(target_sls):
+    sls_file_on_minion = __salt__['cp.cache_file']('{0}{1}.sls'.format('salt://actionchains/', target_sls.replace('actionchains.','')))
+    current_state_info = _read_sls_file(sls_file_on_minion)
+    list_param = current_state_info['schedule_next_chunk']['mgrcompat.module_run']
+
+    for dic in list_param:
+        if 'reboot_required' in dic:
+            return dic["reboot_required"]
+    return False
+
 def _read_next_ac_chunk(clear=True):
     '''
     Read and remove the content of '_mgractionchains.conf' file. Return the parsed YAML.
     '''
     f_storage_filename = _get_ac_storage_filenamepath()
-    if not os.path.isfile(f_storage_filename):
+    ret = _read_sls_file(f_storage_filename)
+    if ret == None:
```

Comment:
Small suggestion, `None` is a singleton (like e.g. `True`/`False`) and singletons should be compared by identity (not by value) according to [PEP8](https://peps.python.org/pep-0008/#programming-recommendations)

```suggestion
    if ret is None:
```